Quadriga/src/test/java/edu/asu/spring/quadriga/service/conceptcollection/mapper/impl/ConceptCollectionShallowMapperTest.java

```diff
@@ -0,0 +1,105 @@
+package edu.asu.spring.quadriga.service.conceptcollection.mapper.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Matchers;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import edu.asu.spring.quadriga.dao.conceptcollection.IConceptCollectionDAO;
+import edu.asu.spring.quadriga.domain.conceptcollection.IConceptCollection;
+import edu.asu.spring.quadriga.domain.impl.User;
+import edu.asu.spring.quadriga.dto.ConceptCollectionDTO;
+import edu.asu.spring.quadriga.dto.QuadrigaUserDTO;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.conceptcollection.IConceptCollectionManager;
+import edu.asu.spring.quadriga.service.user.mapper.IUserDeepMapper;
+
+public class ConceptCollectionShallowMapperTest {
+
+	@Mock
+	private IConceptCollectionDAO mockeddbConnect = Mockito.mock(IConceptCollectionDAO.class);
+
+	@Mock
+	private IConceptCollectionManager mockedccManager = Mockito.mock(IConceptCollectionManager.class);
+
+	@Mock
+	private IUserDeepMapper mockedUserDeepMapper = Mockito.mock(IUserDeepMapper.class);
+
+	@InjectMocks
+	ConceptCollectionShallowMapper conceptCollectionShallowMapperUnderTest;
+
+	@Before
+	public void setUp() {
+		MockitoAnnotations.initMocks(this);
+	}
+
+	@Test
+	public void getConceptCollectionListTest() throws QuadrigaStorageException {
+		QuadrigaUserDTO userDTO = new QuadrigaUserDTO();
+		userDTO.setUsername("username");
+		ConceptCollectionDTO dto = new ConceptCollectionDTO();
+		dto.setCollectionname("collectionname");
+		dto.setOwner(userDTO);
```

Comment:
same here; you could do that in the setup method and use the same object in the other test method